doc/guides/collaborator-guide.md

```diff
@@ -143,30 +143,27 @@ the TSC meeting agenda.
 
 ### Waiting for Approvals
 
-Before landing pull requests, allow 48 hours for input from other Collaborators.
-Certain types of pull requests can be fast-tracked and may land after a shorter
-delay. For example:
+Before landing pull requests, allow for input from other Collaborators.
```

Comment:
Collaborators that are in different timezones or away on local bank holidays... that's why the 48 hour rule exists.

This "one approval rule" is basically a 1-2 for collaborators from the same timezone.